k9mail/src/main/res/xml/account_settings_preferences.xml

```diff
@@ -479,6 +479,12 @@
             android:key="crypto_key"
             android:title="@string/account_settings_crypto_key" />
 
+        <CheckBoxPreference
+            android:key="autocrypt_prefer_encrypt"
```

Comment:
I'd rather setting was either completely crypto agnostic or not. i.e. just "prefer_encrypt"

Reply:
This is not a generic setting - unlike any other, the semantics are precisely defined by autocrypt. Hence the name